### PR TITLE
fixed radio btn bug - wooohooo

### PR DESCRIPTION
**Pull request by katrin-freihofner**
https://github.com/Dynatrace/css-groundhog/issues/27

### CODE REVIEW

File: src/radiobutton/radiobutton.scss [hunk 1]
```diff
@@ -13,6 +13,9 @@
 }
 
 .radio__label:before {
+  margin: 0;
+  padding: 0;
+  top: 0;
```
Comment (tomheller) [inline src/radiobutton/radiobutton.scss]: can you move the top and left rules together for clarity?

**Comment by tomheller** [comment]
Can you bump up the version number in `package.json` to `0.1.1` ?

**Comment by tomheller** [comment]
With a system-font fallback the radios are kind of misaligned in firefox. 
screen shot 2016-08-24 at 07 55 13

File: src/radiobutton/samples/00-standard.html [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+
+<div class="">
+
+<input type="radio" class="radio" id="r1"/>
+<label for="r1" class="radio__label">
+  <span class="radio__caption">Test</span>
+</label>
+
+</div>
+</fieldset>
```
Comment (ddprrt) [inline src/radiobutton/samples/00-standard.html]: There are now two buttons with the same ID. This won't work usually. If you want to make them part of the same group, use `name="group1"` or similar.
Also wondering about the `div`s

**Comment by katrin-freihofner** [comment]
I committed this by accident - was just a test.

**Pull request merged**